src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/ProgressRing/WinUIProgressRing_CustomSources.xaml

```diff
@@ -7,7 +7,8 @@
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 	xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
 	xmlns:lottie="using:Microsoft.Toolkit.Uwp.UI.Lottie"
-    mc:Ignorable="d"
+	xmlns:not_win="http://uno.ui/not_win"
+    mc:Ignorable="d not_win"
```

Comment:
Why are we making changes to ProgressRing samples in this PR?

Reply:
This API was experimental and was removed in subsequent MUX packages unfortunately. We still support it though in Uno targets.